pkg/templates/node_v1.8.go

```diff
@@ -0,0 +1,378 @@
+/* vim: set filetype=yaml : */
```

Comment:
The node 1.7 and 1.8 template look very similar is it currently possible to extract the common stuff and merge in the kubernetes version? 
Changes to the common coreos setup would apply to all kubernetes versions.

Reply:
Yeah, I thought about that too. I'd do it in a separate PR.